microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,102 +3,212 @@ description: The latest experimental features in Microsoft Edge DevTools
+<!-- ====================================================================== -->
+## Enable back forward cache debugging support
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Timeline: event initiators
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Timeline: WebGL based flamechart
+
+Placeholder.
 
-:::image type="content" source="../media/experiments-source-order-viewer.msft.png" alt-text="Source Order Viewer in the Accessibility pane" lightbox="../media/experiments-source-order-viewer.msft.png":::
 
-This experiment is available starting with Microsoft Edge version 86 and is turned on by default.
+<!-- ====================================================================== -->
+## WebAssembly Debugging: Enable DWARF support
+
+Placeholder.
 
 
 <!-- ====================================================================== -->
-## Enable new Font Editor tool within the Styles pane.
-<!-- keep the period per the ui string literal -->
-<!-- found in Experiments in 94 60% down main list -->
+## Enable automatic contrast issue reporting via the Issues panel
 
-You can now use the new visual [Font Editor](../inspect-styles/edit-fonts.md) to edit fonts.  Use it define fonts and font characteristics.  The visual **Font Editor** helps you do the following:
+Placeholder.
 
-*   Switch between units for different font properties
-*   Switch between keywords for different font properties
-*   Convert units
-*   Generate accurate CSS code
 
-To use the new visual **Font Editor**:
-1.  Open the **Elements** tool.
-1.  Open the **Styles** pane.
-1.  Select the **Font Editor** icon.
+<!-- ====================================================================== -->
+## Enable experimental cookie features
+
+Placeholder.
 
-For more information about the new visual **Font Editor**, navigate to [Edit CSS font styles and settings in the Styles pane in DevTools](../inspect-styles/edit-fonts.md).
 
-:::image type="complex" source="../media/font-editor-open.msft.png" alt-text="The visual Font Editor pane is highlighted" lightbox="../media/font-editor-open.msft.png":::
-   The visual **Font Editor** pane is highlighted
-:::image-end:::
+<!-- ====================================================================== -->
+## Enable experimental hide issues menu
 
-This experiment is available starting with Microsoft Edge version 89.
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Allow grouping and hiding of issues by IssueKind
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Enable Reporting API panel in the Application panel
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Enable CSS \<length\> authoring tool in the Styles pane
```

Comment:
@hxlnt - In another PR, you commented "I don't think we picked this up from upstream."  Not sure if there's suggested action here.  The goo.gle link is in the checkbox label in 96 but not later.  We have determined that this page needs to cover the latest Canary checkboxes, each release.  Therefore, the h2 heading shouldn't mention the goo.gle link (because that's been removed from the checkbox label in the latest releases).

Undecided whether the body of this h2 section should show that link, or comment it out (keeping it for authoring), or omit it.

Reply:
In Edge 96, we removed this capability but the (nonfunctional) checkbox remains. This will be removed in subsequent versions of Edge. I think we can just remove this entry entirely.

Reply:
removed (commented out) in the file.